packages/bsky/proto/bsky.proto

```diff
@@ -1393,3 +1396,15 @@ message ClearThreadMutesRequest {
 }
 
 message ClearThreadMutesResponse {}
+
+// Subscription entitlement
+message SubscriptionEntitlement {
+  string did = 1;
+  repeated string entitlements = 2;
```

Comment:
Thoughts on adding a `updated_at` field here? Could be useful in the future for crons that sweep for outdated data.